system/pages/serverinfo.php

```diff
@@ -56,22 +56,22 @@
 if(isset($whiteSkullTime))
     $whiteSkullTime = eval('return ' . $whiteSkullTime . ';');
 
-$redSkullLength = isset($config['lua']['redSkullLength']) ? $config['lua']['redSkullLength'] : NULL;
+$redSkullLength = isset($config['lua']['redSkullDuration']) ? $config['lua']['redSkullDuration'] : NULL;
```

Comment:
nós podemos usar a variável configLua('redSkullDuration') e utilizar a seguinte condição:
```suggestion
$redSkullLength = $configLua['redSkullDuration'] ?? null;
```